nx584/api.py

```diff
@@ -69,13 +69,13 @@ def command():
     args = flask.request.args
     if args.get('cmd') == 'arm':
         if args.get('type') == 'stay':
-            CONTROLLER.arm_stay()
+            CONTROLLER.arm_stay(args.get('partition'))
```

Comment:
To avoid breakage with older clients that expect arming means partition 1, this should be something like:

`args.get('partition', 1)`

and you should do the server-side type conversion, here, so wrap this in int() instead of the other places you have it.

Also, this is an API change, so you need to bump what the `/version` returns to 1.2 to indicate that per-partition arming is supported. Technically, the client should also probably check `/version` and refuse to try to arm if `partition` is anything other than `1` if the server is too old. However, you need to expose it so that at least homeassistant can check it and do something helpful if it's too old.